### ISSUE TITLE
Looks badly proportioned in browser(Chrome).

### ISSUE THREAD

**Issue by taiyoslime** [chubin/wttr.in#12]
## overview
When I access using browser, the layout is badly proportioned, like this capture.
(even `pre` tag and `font-family:monospace`has been already enable.)

![fireshot capture 5 - - http___wttr in_](https://cloud.githubusercontent.com/assets/11515982/13196475/0ee48a06-d812-11e5-855b-c04056b60b9c.png)

##### env
[OS] OS X Yosemite v10.10.2
[browser] Google Chrome v48

**Comment by gshguru** [comment]
Same here. But distortion is very little compared to @taiyoslime's one;
![image](https://cloud.githubusercontent.com/assets/3347300/13197600/92c48ab4-d818-11e5-9f48-ceaa5b850a1c.png)

Not only with chrome, but also in Firefox. 


OSX 10.10.5 
Chrome 48.0.2564.109
Firefox ESR 38.6.0

**Comment by Fenslett** [comment]
@taiyoslime Looks like your browser font is not *a monospaced font* whose every characters take up the same amount of horizontal space. Change the default font like Monaco, Menlo, Courier and so on.

**Comment by schachmat** [comment]
This can be fixed: Serve a sane font like dejavu sans mono from the server.

**Comment by chubin** [comment]
Yes @schachmat , it's really so. That is exactly the way how the problem must be fixed (and will be fixed)
